### DISCUSSION TITLE
Random selection of records (search)

### DISCUSSION THREAD

**Discussion by VTyrnovsky** [TriliumNext/Trilium#4582] [Q&A] [answered]
Colleagues, thank you very much for the trilium notes.
When working with records I would like to implement the following scenario:
1 Select "directory" (parent entry) or several directories.
2 Based on the records inside, create a random sample of 10 records

Please tell me how to make such a selection? Is it possible?

**Comment by Nriver** [comment]
It can be done by using ETAPI with https://github.com/Nriver/trilium-py.

**Reply by VTyrnovsky** [reply]
Do I understand correctly that trilium-py will help solve the problem if I install trilium on a separate server?

In my case, I am currently using an application from flathub

**Reply by Nriver** [reply, marked as answer]
It does not matter where you run trilium. But it does require some Python programming knowledge.
